### PR TITLE
New qb-fix 

### PR DESCRIPTION
**Pull request by Mustachedom**
a few things outside of jim-bridge will need to be updated for qb-inventory to fully work. Registering shops has to be registered server side like ox_inventory but uses a slightly different format. I pasted the code in the discord channel.

has been tested by myself, and 5 others with no issue using new qb-inventory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -539,67 +531,67 @@ stopAnim(
...
+- Loads the model before spawning
+- Unloads model from memory cache after spawn
+- Several options for creation
+- Can spawn with scenario name or anims
+- Spawns invincible
+
+Example of use:
+```lua
+local ped = makePed(
+	`MP_M_Freemode_011,
```
Comment (DeeRock94) [inline README.md:562]: Not your fault, I respect it was already there but let's fix this documentation once and for all

```suggestion
	`MP_M_Freemode_011`,

```

**Comment by DeeRock94** [comment]
Forgive me, I'm slightly new to this repository. Is the presumption here that for backwards compatibility, if a user is still on the _old_ qb-inventory, i.e without the `qb-` prefix for events, that they'll now be _technically_ using the `PSInv` as their inventory? I know PSInv is based heavily on the original QB Inventory structure i.e without the `qb-` prefix.

Presumambly if that's the case, anyone using the new QB inventory will just continue to use Qb when configuring it from the child script config that inherits from this bridge?

**Comment by Mustachedom** [comment]
unfortunately there is no way to make qb-inventory old work properly with this as we make calls to check what script is started so we will urge people to use ps/lj as because qb changed alot unless... 
gimme an hour i may have a solution in my brain but itll take some time and downgrading one of my locals

**Comment by Mustachedom** [comment]
as for the fix you pushed in the readme, i dont know if i can do a PR on this PR but if you submit a PR to my fork, ill pull it and then repush this

**Pull request closed without merging**